### PR TITLE
Implements multiple changes for orc8r to be correctly deployed

### PR DESCRIPTION
**Pull request by gruyaume**
- Adds service registry environment variable to all orc8r services to correctly point to right service registry (k8s)
- Removes unused HELM_RELEASE_NAME environment variable from multiple services
- Adds action to create an Orchestrator Admin User (orchestrator charm)
- Adds action to create an NMS Admin User (nms-magmalte charm)
- Adds certifier relation to orchestrator charm
- Adds kubernetes labels, selectors and annotations so that traffic correctly routes to services
- Adds build-package to charmcraft.yaml because wheels aren't consumed by charmcraft anymore
- Fixes wrong port for directoryd service
- Updates kubernetes_service_patch.py to latest version from upstream lib
- Temporarily removes integration tests because of recurring run failures
- Improves documentation to properly deploy the set of charms in orc8r

### CODE REVIEW

File: orchestrator-bundle/corefile.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  Corefile: |
+    .:53 {
```
Comment (sanchezfdezjavier) [inline orchestrator-bundle/corefile.yml]: Sorry for the ignorance, but what is this `.:53` exactly?

Comment (gruyaume) [reply orchestrator-bundle/corefile.yml]: Ahh I wanted to remove this file. But for your information, this configmap is a config file that is used by the coredns service. This service is the DNS service deployed in your kubernetes cluster (under the kube-system namespace). You can read more about this here: https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/

Comment (gruyaume) [reply orchestrator-bundle/corefile.yml]: Just removed the file

File: orchestrator-bundle/nms-nginx-proxy-operator/README.md [hunk 2]
```diff
@@ -38,6 +38,17 @@ juju deploy magma-nms-magmalte nms-magmalte
 juju relate nms-nginx-proxy nms-magmalte
 ```
 
+## Load Balancer
+
+For NMS to be accessible from outside the Kubernetes cluster, you must enable the `metallb` loadbalancer
+in your microk8s cluster.
+```bash
+microk8s enable metallb
```
Comment (sanchezfdezjavier) [inline orchestrator-bundle/nms-nginx-proxy-operator/README.md:46]: From what I read [here](https://microk8s.io/docs/addon-metallb) this is just a load balancer that is managing the traffic from the `microk8s` node to the machine that is running on. Please, correct me if I'm wrong

Comment (gruyaume) [reply orchestrator-bundle/nms-nginx-proxy-operator/README.md:46]: Each cloud have their own load balancing service (ex. ELB in AWS) but if you run on baremetal and you want to specify the range of IP's that services can be reached out from and have some load balancing capability, metallb is a good option and it's built-in microk8s. Here's the services exposed via LoadBalancers in Magma's AWS deployment:

```
guillaume@thinkpad:~$ kubectl get services -n orc8r | grep Load
fluentd                         LoadBalancer   172.20.69.90     a2cb8a84ed4744e7da7506ea19ac6798-178810149.us-east-1.elb.amazonaws.com    24224:30315/TCP                                            12d
nginx-proxy                     LoadBalancer   172.20.228.141   af250d5ebe33146db9bb7a1fd76f8ac0-1275270732.us-east-1.elb.amazonaws.com   443:32006/TCP                                              12d
orc8r-bootstrap-nginx           LoadBalancer   172.20.141.61    a0e618dc040514c5fa22ca35b3994d1a-594597981.us-east-1.elb.amazonaws.com    80:32737/TCP,443:30426/TCP,8444:31764/TCP                  12d
orc8r-clientcert-nginx          LoadBalancer   172.20.217.83    ad6111c35a8604a768e96b078eacee6d-46878223.us-east-1.elb.amazonaws.com     80:32426/TCP,443:32583/TCP,8443:31792/TCP                  12d
orc8r-nginx-proxy               LoadBalancer   172.20.74.119    ab1bd03a106454883ab76860f8fd6ff3-983977431.us-east-1.elb.amazonaws.com    80:30751/TCP,8443:31724/TCP,8444:32294/TCP,443:30666/TCP   12d
```

File: orchestrator-bundle/orc8r-bundle/README.md [hunk 3]
```diff
@@ -40,14 +46,213 @@ juju bootstrap microk8s microk8s-localhost
...
+juju relate orc8r-subscriberdb-cache postgresql-k8s:db
+juju relate orc8r-subscriberdb postgresql-k8s:db
+```
+
+### DNS Resolution
+Services are to be accessed via domain names. To do so, you will need to map certain addresses to 
+their LoadBalancer IP addresses. Retrieve the IP addresses given to your services.
+
+```bash
+ubuntu@thinkpad:~$ microk8s.kubectl get services -n <your model name> | grep LoadBalancer
```
Comment (sanchezfdezjavier) [inline orchestrator-bundle/orc8r-bundle/README.md]: Should this `ubuntu@thinkpad:~$` be here?

Comment (gruyaume) [reply orchestrator-bundle/orc8r-bundle/README.md]: I can remove it, but ultimately it shows that we're in a unix shell.

**Review by sanchezfdezjavier — APPROVED** [review]
Thanks for this huge PR 🎉

File: orchestrator-bundle/nms-nginx-proxy-operator/README.md (hunk 2 above)
Comment (sanchezfdezjavier) [reply orchestrator-bundle/nms-nginx-proxy-operator/README.md:46]: Thanks, nice explanation

File: orchestrator-bundle/orc8r-bundle/README.md (hunk 3 above)
Comment (sanchezfdezjavier) [reply orchestrator-bundle/orc8r-bundle/README.md]: No, makes sense. I just wanted to make sure you were aware you pasted the whole thing including the prompt

**Pull request merged**